olive/model/hf_onnx_config.py

```diff
@@ -0,0 +1,107 @@
+from typing import Any, Mapping, Optional, OrderedDict
+
+from transformers import PreTrainedTokenizer, TensorType
+from transformers.onnx import OnnxConfigWithPast
+
+ADDITIONAL_MODEL_TYPES = {
```

Comment:
Could  you help add a few comments to tell what the element is ?
Like "model“: feature list, onnx config class?

Reply:
Also quick question: the `feature` looks like the tasks which current pretrained model can support.

Reply:
Yes, I can add some comments. 
That's right, the features are the tasks supported by the model. But transformers.onnx calls them differently. optimum doesn't anymore. That's also why I created the map from tasks to feature so the user doesn't need to provide it themselves unless needed.

Reply:
done!